# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/DocumentChannelChild.cpp
# Commit: 3998aeb0935c
# Full Hash: 3998aeb0935c84872b9eb63e74cc5563f8736f6b
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-12 14:56:55
# Regressor Bug: 1641737
# File Overlap Count: 1
# Description:
#   Bug 1641737 - P5. Remove unnecessary method. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78210
# ==============================================================================

diff -r f6c685e4b841 -r 3998aeb0935c netwerk/ipc/DocumentChannelChild.cpp
--- a/netwerk/ipc/DocumentChannelChild.cpp	Fri Jun 12 05:56:39 2020 +0000
+++ b/netwerk/ipc/DocumentChannelChild.cpp	Fri Jun 12 05:56:46 2020 +0000
@@ -125,13 +125,6 @@
   return IPC_OK();
 }
 
-IPCResult DocumentChannelChild::RecvDeleteSelf() {
-  // This calls NeckoChild::DeallocPGenericChannel(), which deletes |this| if
-  // IPDL holds the last reference.  Don't rely on |this| existing after here!
-  Send__delete__(this);
-  return IPC_OK();
-}
-
 IPCResult DocumentChannelChild::RecvRedirectToRealChannel(
     RedirectToRealChannelArgs&& aArgs,
     nsTArray<Endpoint<extensions::PStreamFilterParent>>&& aEndpoints,